docs/guides/app_package_guide/index.rst

```diff
@@ -20,6 +20,7 @@ The ``build`` property is used to instruct the resolver during the build. For th
 * ``index``: The index file used by opensphere-build-index_.
 * ``scss``: The root SCSS file for the application.
 * ``gcc``: Additional instructions for the `Google Closure Compiler`_.
+* ``moduleDefines``: ``goog.define`` properties that should be resolved to a ``node_module`` path in uncompiled mode.
 
```

Comment:
I'm ecstatic that the guide was updated along with the code. However, shouldn't this be documentation as part of the resolver project?

Reply:
If the resolver had RTD guides, I would say yes. The documentation in the readme is very minimal and this seemed to be the better place for it.

Reply:
I agree and will open a ticket for some more extensive documentation there.